docs/changelog.rst

```diff
@@ -7,7 +7,7 @@ Versions follow `Semantic Versioning <https://semver.org/>`_ (``<major>.<minor>.
 Backward incompatible (breaking) changes will only be introduced in major versions
 with advance notice in the **Deprecations** section of releases.
 
-.. include:: _draft.rst
+.. towncrier-draft-entries:: DRAFT
```

Comment:
@gaborbernat `DRAFT` here is a string argument that is passed to `towncrier --draft --version '{THIS VALUE}'`. It can be anything. For example, it could be `|release| (EXPLANATION THAT IT'S A DRAFT)` and then it'd be rendered like `3.88.99 (EXPLANATION THAT IT'S A DRAFT)`.

I tried to keep the behavior as it is now but you have options FYI.